client/blocks/image-editor/image-editor-canvas.jsx

```diff
@@ -126,18 +126,22 @@ class ImageEditorCanvas extends Component {
 
 		this.drawImage();
 		this.updateCanvasPosition();
-		this.onWindowResize = throttle( this.updateCanvasPosition, 200 );
+
 		if ( typeof window !== 'undefined' ) {
 			window.addEventListener( 'resize', this.onWindowResize );
 		}
 
 		this.props.setImageEditorImageHasLoaded( this.image.width, this.image.height );
 	}
 
+	onWindowResize() {
+		this.requestAnimationFrameId = window.requestAnimationFrame( this.updateCanvasPosition );
```

Comment:
> is client/libs the best place for new utils?

Yes, that would be a good spot for any hypothetical general use libs.

>  I was going to implement something similar to this

Ah, I think we might have had that approach in the first implementation? So the only difference with my suggestion would be to limit how often the calculations run. On a fast computer we'll basically be calculating frames as fast as we can get through them, when we can maybe target a max of 30 or 60 fps.